.changes/unreleased/NOTES-20250210-175414.yaml

```
@@ -0,0 +1,5 @@
+kind: NOTES
+body: 'tfprotov5+tfprotov6: An upcoming release will require the `ProviderServerWithResourceIdentity` implementation as part of `ProviderServer`.'
```

nit: maybe a little more specific
```suggestion
body: 'tfprotov5+tfprotov6: An upcoming release will require the `GetResourceIdentitySchemas` and `UpgradeResourceIdentity` implementations as part of `ProviderServer`.'
```